input/profiles/StructureDefinition-qicore-medicationnotrequested.json

```diff
@@ -71,6 +71,26 @@
         ],
         "mustSupport": false
       },
+	 {
+			"id" : "MedicationRequest.medicationAdherence",
+			"extension" : [
+			  {
+				"url" : "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-keyelement",
+				"valueBoolean" : true
+			  }
+			],
+			"path" : "MedicationRequest.extension",
+			"sliceName" : "medicationAdherence",
+			"short" : "URL: http://hl7.org/fhir/us/core/StructureDefinition/us-core-medication-adherence",
```

Comment:
This url for us-core-medication-adherence shouldn't be as short, which rendered as plain text. It should be rendered as a hyperlink that take to the us-core-medication-adherence extension.